.gitignore

```diff
@@ -40,3 +40,5 @@ log.txt
 **/_site
 **/.sass-cache
 **/.jekyll-metadata
+
+mesheryctl/.meshery/
```

Comment:
@girishranganathan can we move to line 3 or so under #Meshery?

Reply:
done